newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -1444,6 +1448,14 @@ const VariablesList = React.forwardRef<Props, VariablesListInterface>(
         props.inheritedVariablesContainer &&
         props.inheritedVariablesContainer.has(name);
 
+      const typeErrorMessage =
+        parentType === gd.Variable.Array &&
+        parentVariable &&
+        parentVariable.getChildrenCount() > 1 &&
+        parentVariable.getAtIndex(0).getType() !== type
+          ? i18n._(t`Array children must all be of the same type.`)
```

Comment:
<img width="778" alt="image" src="https://github.com/user-attachments/assets/1e880ce8-1329-45da-84a6-e0c40e3e7653" />

Reply:
- "children" is better than "elements" in our case as it's how we call them.
- "must" is already stronger than "should" so I see no reason to change it.
- "for consistency" is just wrong, it's not about making it easier to understand, it's about making it work.

I think it's a good case of bad usage of AI. If you ask it to paraphrase it will, not always for the best.